python/ray/tune/tests/execution/test_controller_checkpointing_integration.py

```diff
@@ -15,15 +15,23 @@
 from ray.train._internal.session import _TrainingResult
 from ray.train._internal.storage import StorageContext
 from ray.train.tests.util import mock_storage_context
-from ray.tune import PlacementGroupFactory, ResumeConfig
+from ray.tune import PlacementGroupFactory, ResumeConfig, register_trainable
 from ray.tune.execution.tune_controller import TuneController
 from ray.tune.experiment import Trial
 from ray.tune.result import DONE
 from ray.tune.schedulers import FIFOScheduler
 from ray.tune.search import BasicVariantGenerator
 from ray.tune.tests.tune_test_util import TrialResultObserver
+from ray.tune.utils.mock_trainable import MyTrainableClass
 
 STORAGE = mock_storage_context()
+MOCK_TRAINABLE_NAME = "mock_trainable"
```

Comment:
This can also be in `ray.tune.utils.mock_trainable`? Instead of defining in each file

Reply:
done